crates/viewer/re_ui/src/design_tokens.rs

```diff
@@ -289,7 +289,7 @@ impl DesignTokens {
     }
 
     pub fn table_line_height() -> f32 {
```

Comment:
We should probably note that this excludes margins

Reply:
It doesn't, though. We don't have any explicit vertical margin, we just center stuff within the line space.